tests/depends/Trie/CMakeLists.txt

```diff
@@ -1,12 +1,14 @@
-# add_executable(Test_Trie Test_Trie.cpp)
-# target_include_directories(Test_Trie PUBLIC ${CMAKE_SOURCE_DIR}/src ${CMAKE_SOURCE_DIR}/tests)
-# target_link_libraries(Test_Trie PUBLIC boost_system TestUtils Trie Utils)
+link_directories(${CMAKE_BINARY_DIR}/lib)
+
+add_executable(Test_Trie Test_Trie.cpp)
+target_include_directories(Test_Trie PUBLIC ${CMAKE_SOURCE_DIR}/src ${CMAKE_SOURCE_DIR}/tests)
+#target_link_libraries(Test_Trie PUBLIC boost_system TestUtils Trie Utils)
```

Comment:
A line of debugging code. Is this still useful?

Reply:
Noted